dotnet/src/SemanticKernel/Planning/Plan.cs

```diff
@@ -133,21 +131,27 @@ public Plan(ISKFunction function)
     /// <param name="description">The description of the plan.</param>
     /// <param name="nextStepIndex">The index of the next step.</param>
     /// <param name="state">The state of the plan.</param>
-    /// <param name="namedParameters">The named parameters of the plan.</param>
-    /// <param name="namedOutputs">The named outputs of the plan.</param>
+    /// <param name="parameters">The parameters of the plan.</param>
+    /// <param name="outputs">The outputs of the plan.</param>
     /// <param name="steps">The steps of the plan.</param>
     [JsonConstructor]
-    public Plan(string name, string skillName, string description, int nextStepIndex, ContextVariables state, ContextVariables namedParameters,
-        ContextVariables namedOutputs,
+    public Plan(
+        string name,
+        string skillName,
+        string description,
+        int nextStepIndex,
+        ContextVariables state,
+        ContextVariables parameters,
+        IList<string> outputs,
         IReadOnlyList<Plan> steps)
     {
         this.Name = name;
         this.SkillName = skillName;
         this.Description = description;
         this.NextStepIndex = nextStepIndex;
         this.State = state;
-        this.NamedParameters = namedParameters;
-        this.NamedOutputs = namedOutputs;
+        this.Parameters = parameters;
```

Comment:
`Outputs` is a much better name but I see you kept the `Parameters` instead of using `Inputs`.

My suggestion just to make it easier for the audience is where I see `Outputs` I would expect an `Inputs`.

`InputParameters` and `OutputParameters` would be my ask if you want to keep the `Parameters`

Reply:
Let's follow up separately on this one. I kept with `Parameters` since that's the naming on `SKFunction`